owlbot.py

```diff
@@ -55,10 +54,6 @@
     ],
 )
 
-# Remove unneeded intersphinx links, the library does not use any proto-generated code.
-s.replace("docs/conf.py", r'\s+"(proto-plus|protobuf)":.*$', "")
```

Comment:
Related to the comment above, we actually want to include proto* intersphinx links.